src/Query/Processor.php

```diff
@@ -42,6 +42,18 @@ public function processSelect(Builder $query, $results): array
         return $results;
     }
 
+    /**
+     * @param list<array{ TABLE_NAME: string }> $results
+     * @return list<string>
+     */
+    public function processGetAllTables(array $results): array
+    {
+        return array_map(
+            static fn(array $result) => $result['TABLE_NAME'],
```

Comment:
```suggestion
            static fn(array $result): string => $result['TABLE_NAME'],
```

Reply:
I usually don't add return types since phpstan should be taking care of type checking.